CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+## Contributing
+
+All pull requests should be made to the develop branch.
+
+```
+git clone https://github.com/vinnymac/PokeNurse
+cd PokeNurse
+git checkout develop
+yarn
```

Comment:
Incoming PRs are few & far between.. with the future of all poke-related apps in jeopardy, I don't know that it's worth spending a lot of time laying out conventions.